sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -3096,12 +3096,7 @@ SIREPO.app.directive('ldapLogin', function (requestSender, errorService) {
         `,
         controller: function ($scope) {
             function handleResponse(data) {
-                if (data.state == 'ok') {
-                    showWarning(data.formError);
-                }
-                else {
-                    showWarning('Server reported an error, please contact support@sirepo.com.');
-                }
+                showWarning(data.error);
```

Comment:
Let's continue to use formError. data.error is too coarse and conflicts with other uses in sirepo.